WORKING_GROUPS/Bridge_PoCs/snippets/hmacgen.py

```diff
@@ -0,0 +1,36 @@
+#https://matrix-org.github.io/synapse/latest/admin_api/register_api.html
+import hmac
+import hashlib
+import requests
+import json
+
+nonce_request = requests.get("https://matrix.blueskycommunity.net/_synapse/admin/v1/register")
+nonce = nonce_request.json()["nonce"]
+
```

Comment:
can you add code to pull variables server_secret, username, pw from environment?

Reply:
done!